src/Akka.Persistence.Redis.Tests/Query/RedisCurrentPersistenceIdsSpec.cs

```diff
@@ -40,11 +40,6 @@ public RedisCurrentPersistenceIdsSpec(ITestOutputHelper output, RedisFixture fix
             ReadJournal = Sys.ReadJournalFor<RedisReadJournal>(RedisReadJournal.Identifier);
         }
 
-        [Fact(Skip = "Not implemented yet")]
```

Comment:
Never delete a code that is not yours, it was there because someone thinks that it is needed, in this case it is a reminder that we have a techincal debt and this test might be needed in the future.

Reply:
Did that since that has been implemented